src/docs/product/integrations/integration-platform/public-integration.mdx

```diff
@@ -0,0 +1,210 @@
+
+```json
+{
+  "id": "38",
+  "token": "ec48bf98637d44c294ead7566513686237e74ab67a074c64b3aaca2d93dbb8f1",
+  "refreshToken": "c866f154a65841638d44ee26364409b0a1a67bd642bd46e7a476f34f810712d6",
+  "dateCreated": "2019-08-07T20:25:09.870Z",
+  "expiresAt": "2019-08-08T04:25:09.870Z",
+  "state": null,
+  "application": null
+}
+```
+
+## Auth Tokens
+
+Auth Tokens are referred to as _tokens_ in the Integration Platform. Auth tokens are passed using an auth header and are used to authenticate as a user account with the API. In the case of the Integration Platform, these tokens are used to authenticate as an application (with access to a specific organization) instead of a user.
+
+
+## Using Tokens for Requests
+
+When making requests to the Sentry API, you use the token just like you would when you're making typical [API requests](/api/auth/). Tokens are associated with the installation, meaning they have access to the Sentry organization that installed your integration.
+
+## Token Expiration
+
+Tokens for public integrations expire every eight hours.
+
+## Refreshing Tokens
+
+The tokens you receive from Sentry expire after eight hours. To retrieve a new token, make a request to the same Authorization endpoint used in the setup endpoint above, but with a slightly different request body:
+
+```python
+def refresh_token(install_id):
+    url = u'https://sentry.io/api/0/sentry-app-installations/{}/authorizations/'
+    url = url.format(install_id)
+
+    refresh_token = retrieve_refresh_token_from_db(install_id)
+
+    payload = {
+        'grant_type': 'refresh_token',
+        'refresh_token': refresh_token,
+        'client_id': 'your-client-id',
+        'client_secret': 'your-client-secret',
+    }
+
+    resp = requests.post(url, json=payload)
+    data = resp.json()
+
+    new_token = data['token']
+    new_refresh_token = data['refreshToken']
+    # ... Securely update the token and refresh_token in DB...
+
+    return new_token
+```
+
+Instead of keeping track of times and passively refreshing when a token expires, another way you can handle refreshing tokens is to actively capture exceptions raised by requests that receive a 401 unauthorized response from Sentry, refresh the token, and remake the request.
+
+### Verifying Installations (optional)
+
+Typically, if you have the redirect URL configured, there is work happening on your end to "finalize" the installation. If this is the case, we recommend enabling the "Verify Install" option for your integration. Once enabled, you'll need to send a request marking the installation as officially "installed":
+
+```python
+requests.put(
+    u'https://sentry.io/api/0/sentry-app-installations/{}/'.format(install_id),
+    json={'status': 'installed'},
+)
+```
+
+
+## Uninstallation
+
+When a user uninstalls your integration, you'll receive a webhook request to your webhook URL.
+
+## Integration Webhooks
```

Comment:
I think this entire section can be removed as well. I think once the reference implementation is built we can just mention it here instead of trying to paste in big code blobs where the intention of the code isn't immediately obvious.